Makefile

```diff
@@ -21,10 +21,10 @@ fmt :
 	$(GO_FMT) -s -w ./
 
 test :
-	go test -race -coverprofile=coverage.txt -covermode=atomic ./router ./msgservice ./dbservice ./formatting ./data ./regoservice ./routes
+	go test -race -coverprofile=coverage.txt -covermode=atomic ./router ./msgservice ./dbservice ./formatting ./data ./regoservice ./routes ./outputs
 
 cover :
-	go test ./msgservice ./dbservice ./router ./formatting ./data ./regoservice ./routes -v -coverprofile=cover.out
+	go test ./msgservice ./dbservice ./router ./formatting ./data ./regoservice ./routes ./outputs -v -coverprofile=cover.out
```

Comment:
Ideally we should just include all directories like so https://github.com/aquasecurity/postee/pull/287/commits/31bda2aad34e34ac7a74d78d50f540304ec3f399

When I did this, I noticed there are two tests that are making external calls, in this commit I disabled them (for now) until we can find a way to better handle them: https://github.com/aquasecurity/postee/pull/287/commits/31bda2aad34e34ac7a74d78d50f540304ec3f399

Reply:
If you want to merge this PR after `cc6c7a5` and `31bda2a` merge, I can remove these changes.